.rubocop.yml

```diff
@@ -37,38 +37,47 @@ Style/Semicolon:
 Style/WordArray:
   MinSize: 5
 
-# Listed below are rules we don't plan to enable in the foreseeable future.
-# Add rules we plan to fix (but are currently failing) to .rubocop_todo.yml.
+# END CODE.ORG OVERRIDES
 
-Style/BarePercentLiterals:
+# BEGIN BLACKLIST
+
+Metrics/AbcSize:
   Enabled: false
 
-Style/ClosingParenthesisIndentation:
+Metrics/BlockNesting:
   Enabled: false
 
-Style/ExtraSpacing:
+Metrics/LineLength:
   Enabled: false
 
-Style/IdenticalConditionalBranches:
+Metrics/ModuleLength:
   Enabled: false
 
-Style/IfInsideElse:
+Style/AsciiComments:
   Enabled: false
 
-Style/RedundantReturn:
+Style/BarePercentLiterals:
   Enabled: false
 
-Metrics/AbcSize:
+Style/ClosingParenthesisIndentation:
   Enabled: false
 
-Metrics/LineLength:
+Style/ExtraSpacing:
   Enabled: false
 
-Metrics/ModuleLength:
+Style/IdenticalConditionalBranches:
   Enabled: false
 
-Metrics/BlockNesting:
+Style/IfInsideElse:
   Enabled: false
 
-Style/AsciiComments:
+Style/RedundantReturn:
   Enabled: false
+
+# As of July 2016, there is enough disagreement in our codebase, on our team,
+# and apathy to the rule to suggest a negative cost-benefit of enabling the
+# rule.
```

Comment:
FWIW I think the existing comment covers the current status:

```yml
# Listed below are rules we don't plan to enable in the foreseeable future.
# Add rules we plan to fix (but are currently failing) to .rubocop_todo.yml.
```

Historical context can be found in these PRs.